metaswitch/common/utils.py

```diff
@@ -401,6 +401,29 @@ def sigusr1_handler(sig, stack):
         write_core_file(process_name, stack_dump)
     signal.signal(signal.SIGUSR1, sigusr1_handler)
 
+def map_clearwater_log_level(level):
+    """
+    Map from Clearwater log levels to Python log levels.
+
+    Python doesn't have status or verbose levels, so these are mapped to
+    info and debug respectively.
+    """
+    LOG_LEVELS = {0: logging.ERROR,
+                  1: logging.WARNING,
+                  2: logging.INFO,
+                  3: logging.INFO,
+                  4: logging.DEBUG,
```

Comment:
etcd has 4 as DEBUG. Given how little we use Verbose, I'm tempted to think it should be debug.